python/Makefile.am

```diff
@@ -16,7 +16,10 @@ SUBDIRS	= pacemaker \
 
 .PHONY: check-local
 check-local:
-	PYTHONPATH=$(top_srcdir)/python:$(top_builddir)/python $(PYTHON) -m unittest discover -v -s $(top_srcdir)/python/tests $(top_builddir)/python/tests
+	if [ "x$(top_srcdir)" != "x$(top_builddir)" ]; then \
+		( cd $(top_srcdir)/python ; find . -name '*py' | cpio -pmd $(abs_top_builddir) ) \
```

Comment:
The cpio man page says it's considered "legacy." Considering we don't have any dependency on it currently, it's probably better to avoid. Plain old cp should work fine for Python source, though you could use `$(LN_S)` if you wanted to make symlinks.